src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -415,7 +415,8 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
                         {
                             FormattedText = formattedText,
                             Title = title
-                        }
+                        },
+                        GuidesManager = this
```

Comment:
This was set to launch the Onboarding Guide once the link of the survey's step is clicked